src/main/java/org/jabref/preferences/GeneralPreferences.java

```diff
@@ -81,6 +87,60 @@ public void setConfirmDelete(boolean confirmDelete) {
         this.confirmDelete.set(confirmDelete);
     }
 
+    /**
+     * Get the preference of whether to give a confirmation dialog when empty entries are detected in the library
+     *
+     * @return true if the preference is set as yes
+     */
+    public boolean shouldConfirmEmptyEntries() {
+        return confirmEmptyEntries.get();
+    }
+
+    /**
+     * Get the BooleanProperty of the preference of whether to give a confirmation dialog when empty entries are detected in the library
+     *
+     * @return BooleanProperty
+     */
+    public BooleanProperty confirmEmptyEntriesProperty() {
+        return confirmEmptyEntries;
+    }
+
+    /**
+     * Set the preference of whether to give a confirmation dialog when empty entries are detected in the library
+     *
+     * @param confirmEmptyEntries        boolean for the preference
+     */
+    public void setConfirmEmptyEntries(boolean confirmEmptyEntries) {
+        this.confirmEmptyEntries.set(confirmEmptyEntries);
+    }
+
+    /**
+     * Get the preference of whether to delete the empty entries detected in the library if confirmation dialog is opted out
+     *
+     * @return true if the preference is set as yes (delete)
+     */
+    public boolean shouldDeleteEmptyEntries() {
+        return deleteEmptyEntries.get();
+    }
+
+    /**
+     * Get the BooleanProperty of the preference of whether to delete the empty entries detected in the library if confirmation dialog is opted out
+     *
+     * @return BooleanProperty
+     */
+    public BooleanProperty deleteEmptyEntriesProperty() {
+        return deleteEmptyEntries;
+    }
+
+    /**
+     * Set the preference of whether to delete the empty entries detected in the library if confirmation dialog is opted out
+     *
+     * @param deleteEmptyEntries        boolean for the preference
+     */
+    public void setDeleteEmptyEntries(boolean deleteEmptyEntries) {
+        this.deleteEmptyEntries.set(deleteEmptyEntries);
+    }
```

Comment:
This javadoc is superfluous. Please comment on the the variable or the property, but not on every getter and setter

Reply:
Sorry about this. I removed these comments. Let me know if you would like have any other changes.